.changeset/hungry-rockets-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': minor
+---
+
+Add custom aria label to TreeView.Item
```

Comment:
Nit-pick: Could we add a little more details into the changeset? 

I.e. "Add an ability to provide custom `aria-label` and `aria-labelledby` to `TreeView.Item`"